### PR TITLE
[14.0][MIG] contract_line_tax

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ connector
 account-invoicing
 account-financial-tools
 l10n-spain
+contract
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: can you put changes to oca_dependencies in a different commit? We need it in a different commit to be able to aggreate all the open PRs without conflicts

Comment (eantones) [reply oca_dependencies.txt]: Yes, I just got a conflict too, @ChrisOForgeFlow can you please split the commit?

**Review by mariadforgeflow — APPROVED** [review]
